src/index.html

```diff
@@ -1,12 +1,265 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="page-content">
+    <header class="page-content__header">
+      <div class="container">
+        <a href="#home" class="linklogo">
+          <img  class="logomoyo" src="./images/logo.png" alt="MOYOimage">
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link nav__link--active is-active" href="#">Apple
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Samsung</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Смартфоны и телефоны</a>
+            </li>
+            <li class="nav__item">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#"
+              >
+               Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Гаджеты</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Планшеты</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Фото</a>
+            </li>
+            <li class="nav__item">
+              <a class="nav__link" href="#">Видео</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="page-content__main">
+      <div class="pc-wrap">
+
+        <div class="product-card product-card--wrap" data-qa="card">
+          <img class="product-card__img" src="./images/imac.jpeg" alt="MacImage">
+          <div class="product-card__description">
+            <p class="product-card__description-model">
+              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+            </p>
+            <p class="product-card__description-id">Код товара: 195434</p>
+          </div>
```

Comment:
Don't create additional blocks
```suggestion
            <p class="product-card__description-model">
              Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
            </p>
            <p class="product-card__description-id">Код товара: 195434</p>
```